ocrd_tesserocr/binarize.py

```diff
@@ -41,37 +48,36 @@ def process(self):
         
         Set up Tesseract to recognize the segment image's layout, and get
         the binarized image. Create an image file, and reference it as
-        AlternativeImage in the element and as file with a fileGrp USE
-        equal `OCR-D-IMG-BIN` in the workspace.
+        AlternativeImage in the segment element. Add the new image file
+        to the workspace with with the fileGrp USE given in the second position
+        of the output fileGrp, or ``OCR-D-IMG-BIN``, and an ID based on input
```

Comment:
Why the double back ticks?

Reply:
Because by [PEP 287](https://www.python.org/dev/peps/pep-0287), docstrings are to be formatted in reStructuredText, if anything. This is also expected by tools like Sphinx.